documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -6,28 +6,18 @@ Release Notes
 
 6.3.11
 ======
-* Added a hint field in the trace event when all replicas of some data are lost. `(PR #4209) <https://github.com/apple/foundationdb/pull/4209>`_
-* Rewrote SQLite injected fault handling. `(PR #4212) <https://github.com/apple/foundationdb/pull/4212>`_
-* Add a SevWarnAlways trace line to help debug a rare failure. `(PR #4214) <https://github.com/apple/foundationdb/pull/4214>`_
-* Use VFSAsyncFile::checkInjectedError to detect injected faults. `(PR #4253) <https://github.com/apple/foundationdb/pull/4253>`_ 
-* Build on Windows using VS 2019 + LLVM/Clang. `(PR #4258) <https://github.com/apple/foundationdb/pull/4258>`_ 
-* RateControl support in AFCCached to enable write op throttling. The feature is disabled by default. `(PR #4229) <https://github.com/apple/foundationdb/pull/4229>`_ 
```

Comment:
Why should we delete this PR 4229 from the release note.
Although it's disabled by a knob by default, SRE can still enable it and play with it. 
In addition, in case someone accidentally enable this by knob, causing unexpected behavior,  having the release note can remind people that there is a feature that may affect a cluster's behavior and that he/she should check if the feature is accidentally enabled.

Reply:
According to Steve, this was implemented in 6.2 as a patch and has a release note there, so we don't need to add one here.